x/emissions/types/params.go

```diff
@@ -14,7 +14,7 @@ func DefaultParams() Params {
 		MinTopicWeight:                  alloraMath.MustNewDecFromString("100"),    // total weight for a topic < this => don't run inference solicatation or loss update
 		MaxTopicsPerBlock:               uint64(128),                               // max number of topics to run cadence for per block
 		RequiredMinimumStake:            cosmosMath.NewInt(100),                    // minimum stake required to be a worker or reputer
-		RemoveStakeDelayWindow:          int64(60 * 60 * 24 * 7 * 3),               // number of seconds to wait before finalizing a stake withdrawal
+		RemoveStakeDelayWindow:          int64(60 * 60 * 24 * 7 * 3 / 5),           // ~approx 3 weeks assuming 5 second block time, number of blocks to wait before finalizing a stake withdrawal
```

Comment:
math like this where the order of operations is potentially ambiguous always creeps me out, but I suppose this is fine. If it were me I would put parentheses around everything except `/ 5`